samples/cpp/tutorial_code/features2D/feature_flann_matcher/SURF_FLANN_search_dataset_Demo.cpp

```diff
@@ -0,0 +1,270 @@
+// SURF_FLANN_search_dataset_Demo.cpp
```

Comment:
> tutorial_code

This code must belong to some tutorial (it should be mentioned by some tutorial).

Reply:
Moved to sample/cpp/